main.py

```diff
@@ -159,74 +156,55 @@ def __main__():
         # module. This combination allows one to run and debug their stix-shifter code locally, while interacting with
         # it inside a service provider such as IBM Security Connect
         app = Flask("stix-shifter")
-        
+
         @app.route('/transform_query', methods=['POST'])
         def transform_query():
-            request_args = request.get_json( force=True )
-            translation = stix_translation.StixTranslation()
-            dsl = translation.translate(args.translation_module, 'query', args.data_source, request_args["query"])
-            return json.dumps(dsl)
+            host = ProxyHost()
+            return ProxyHost.transform_query(host)
 
         @app.route('/translate_results', methods=['POST'])
         def translate_results():
-            request_args = request.get_json( force=True )
-            translation = stix_translation.StixTranslation()
-            print( json.dumps(request_args["results"]))
-            dsl = translation.translate(args.translation_module, 'results', args.data_source, request_args["results"])
-            return json.dumps(dsl)
+            data_source_identity_object = args.data_source
+            host = ProxyHost()
+            return ProxyHost.translate_results(host, data_source_identity_object)
```

Comment:
yurri already pointed out about this syntax. can we do this- `return host.translate_results(host, data_source_identity_object)`  as the `host` object of `ProxyHost()` class has been created just before that. And also i think we don't need to pass `host` again as an argument. so the final statement should be-

`return host.translate_results(data_source_identity_object)`